corehq/apps/accounting/migrations/0094_add_annual_softwareplans.py

```diff
@@ -9,6 +9,15 @@ def _bootstrap_new_annual_pricing(apps, schema_editor):
     ensure_plans(BOOTSTRAP_CONFIG, verbose=True, apps=apps)
 
 
+def _remove_annual_default_product_plans(apps, schema_editor):
```

Comment:
I don't know if this is strictly necessary but was easy enough to add, so that _in case_ it needs to be reverted for any reason we won't be stuck doing so manually on every environment.